Salesforce/Model/Datasource/SalesforceSource.php

```diff
@@ -221,11 +221,7 @@ public function create(Model $model, $fields = null, $values = null) {
          * @return array|mixed
          */
         public function read(Model $model, $queryData = array(), $recursive = null) {
-            $limit = "";
```

Comment:
By removing this how are you handling limits? Old school CakePHP coding I know but at the time Salesforce required a limit or it returned the lot - has this changed?

Reply:
The limit is there, just moved on the bottom of method as `LIMIT` is always appended to `$sfq` when it is present. I just get rid of `$limit` variable and append directly (same with `OFFSET`)

Reply:
Ill happily trust you on this - I cant possibly test it due to not having this project running anymore - things look fine :)

Thanks for your contribution - much appreciated :)